experiments/OceanBoxGCM/simple_box.jl

```diff
@@ -68,7 +69,7 @@ function run_simple_box(
     grid = driver_config.grid
     vert_filter = CutoffFilter(grid, polynomialorder(grid) - 1)
     exp_filter = ExponentialFilter(grid, 1, 8)
-    modeldata = (vert_filter = vert_filter, exp_filter = exp_filter)
+    modeldata = (vert_filter = vert_filter, exp_filter = exp_filter, mymodeldata = mymodeldata )
```

Comment:
What is `modeldata` and it's purpose? It's hard for me to interpret whether `mymodeldata` is the right name for this arg (I'm suspicious of putting the prefix `my` in front of anything) but otherwise looks fine.

Reply:
You could just all it `modeldata` and splat it onto the tuple (though then you could end up with naming problems if duplicates are used)
```suggestion
    modeldata = (vert_filter = vert_filter, exp_filter = exp_filter, modeldata... )
```

Reply:
@jkozdon I was thinking it might be good to experiment with having a private namespace. I am not sure exactly yet! 

@glwagner modeldata is a tuple that holds a bunch of metadata that is then carried around with the DG abstraction. It is mean to used/abused as someone sees fit. Right now I want to use it to pass an option into writevtk ( https://github.com/CliMA/ClimateMachine.jl/blob/6c97b3f16e238893bb218f0036a5e1908ba468ee/src/InputOutput/VTK/writevtk.jl#L63  ) that would let me add a feature so that this range here 

https://github.com/CliMA/ClimateMachine.jl/blob/6c97b3f16e238893bb218f0036a5e1908ba468ee/src/InputOutput/VTK/writevtk.jl#L133

could be customized to run from say -0.9 to 0.9 or whatever range I am interested in (and/or some other meta things like only doing VTK for some elements for a particular callback etc....). 

I am thinking of this as a convenience for little things that may not become standard, but can be useful.

Reply:
I like @jkozdon's suggestion!

Reply:
> @jkozdon I was thinking it might be good to experiment with having a private namespace. I am not sure exactly yet!

Makes sense.